client/post-editor/editor-mobile-navigation/index.jsx

```diff
@@ -23,17 +57,26 @@ const EditorMobileNavigation = React.createClass( {
 
 		return (
 			<div className="editor-mobile-navigation">
-				<Site indicator={ false } site={ this.props.site } />
-				<button className="button editor-mobile-navigation__toggle" onClick={ this.toggleSidebar }>
-					{ this.translate( 'Actions' ) }
-				</button>
-				<Gridicon
-					icon="cross"
-					onClick={ this.props.onClose }
-					className="editor-mobile-navigation__close" />
+				<div className="editor-mobile-navigation__actions">
+					<Gridicon
+						icon="chevron-left"
+						onClick={ this.props.onClose }
```

Comment:
Semantically, if these are action items, they should be proper buttons, rather than applying a click handler to the icon element directly. Take a look at [the `<Button borderless />` component](https://github.com/Automattic/wp-calypso/tree/master/client/components/button), as it may be appropriate here. Additionally, while I'm not sure the status of screen-readers for mobile devices, this is not very accessible. In the `<Button />` component we may also consider including a `<span className="screen-reader-text" />` with labels describing the actions.